queries.sql

```diff
@@ -1,25 +1,176 @@
-/*Queries that provide answers to the questions from all projects.*/
+/*Queries that provide answers to the questions from all projects.*/ -- Find all animals whose name ends in "mon".
 
--- Find all animals whose name ends in "mon".
-SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT *
+FROM animals
+WHERE name LIKE '%mon';
 
 -- List the name of all animals born between 2016 and 2019.
-SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name
+FROM animals
+WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
 
 -- List the name of all animals that are neutered and have less than 3 escape attempts.
-SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+
+SELECT name
+FROM animals
+WHERE neutered = true
+     AND escape_attempts < 3;
 
 -- List the date of birth of all animals named either "Agumon" or "Pikachu".
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT date_of_birth
+FROM animals
+WHERE name IN ('Agumon',
+               'Pikachu');
 
 -- List name and escape attempts of animals that weigh more than 10.5kg.
-SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT name,
+       escape_attempts
+FROM animals
+WHERE weight_kg > 10.5;
 
 -- Find all animals that are neutered.
-SELECT * FROM animals WHERE neutered = true;
+
+SELECT *
+FROM animals
+WHERE neutered = true;
 
 -- Find all animals not named Gabumon.
-SELECT * FROM animals WHERE name <> 'Gabumon';
+
+SELECT *
+FROM animals
+WHERE name <> 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (inclusive).
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+
+SELECT *
+FROM animals
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+---------------Change species and ROLLBAKC the CHANGES-------------------
+ BEGIN;
+
+
+UPDATE animals
+SET species = 'unspecified'
+SELECT *
+FROM animals;
+
+
+ROLLBACK;
+
+
+SELECT *
+FROM animals;
+
+--------- Update the table and Commit the transaction...........
+ BEGIN;
+
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+
+SELECT *
+FROM animals
+WHERE name LIKE '%mon';
+
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+
+SELECT *
+FROM animals
+WHERE species = 'pokemon';
+
+
+COMMIT ---------------DELETE  animals table and ROLLBACK---------
```

Comment:
* As mentioned in the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20changes,persist%20after%20commit.) you are to 
* Verify that changes were made.
* Commit the transaction then,
* Verify that changes persist after commit. Kindly write queries for these checks before and after the comit